app/views/wands/show.html.erb

```diff
@@ -32,7 +32,7 @@
     <%= link_to 'delete', wand_path, method: :delete, data: { confirm: "Are you sure?" } %>
   <% end %>
 <div class="container col-sm-8 text-center">
-  <button type="button" class="btn btn-secondary btn-lg btn btn-light mb-20">
+  <button type="button" class="btn btn-secondary btn-lg btn button">
```

Comment:
I think we should rename this button something more specific, like show-page-button 
you don't have to rename the file, as we can use that file to put all button partials in, but I think just so we know where things are connected and can follow easier we should name it more specifically

Reply:
Ok. I can rename it. I was thinking that we would make all our buttons look the same throughout the app